mysql-persistence/src/main/java/com/netflix/conductor/dao/mysql/MySQLWorkflowModule.java

```diff
@@ -42,9 +42,10 @@ public DataSource getDataSource(Configuration config) {
 
     @Override
     protected void configure() {
-        bind(MetadataDAO.class).to(MySQLMetadataDAO.class);
-        bind(ExecutionDAO.class).to(MySQLExecutionDAO.class);
-        bind(QueueDAO.class).to(MySQLQueueDAO.class);
+        
+        bind(MetadataDAO.class).to(MySQLMetadataDAO.class).asEagerSingleton();
+        bind(ExecutionDAO.class).to(MySQLExecutionDAO.class).asEagerSingleton();
+        bind(QueueDAO.class).to(MySQLQueueDAO.class).asEagerSingleton();
```

Comment:
Was this solving a booting problem you were running into?

I'm not sure of the implications of `asEagerSingleton()` with respect to the rest of the modules. This will likely require feedback from @v1r3n or someone with more overall knowledge of the module bootstrapping process to weigh in.